src/globals/scss/core/_links.scss

```diff
@@ -63,4 +63,18 @@
       color: $govuk-black;
     }
   }
+
+  // 'Unvisited' link variant
+  //
+  // Used in cases where it is not helpful to distinguish between visited and
+  // non-visited links.
+  //
+  // For example, navigation links to pages with dynamic content like admin
+  // dashboards. The content on the page is changing all the time, so the fact
+  // that you’ve visited it before is not important.
+  .govuk-link--unvisited {
+    &:visited {
```

Comment:
Can we leave this unnested?

Reply:
We can, though I think there is some value in being consistent with the way the rest of the file is written, making it easier to scan. I also personally think it'd be easy to miss if it was on the same line as the class name.

Reply:
Ah sorry, I didn't see the rest of the file in this diff.

Sounds good to me 👍